### ISSUE TITLE
almost duplicate "it is clear" line in statement0.2.md

### ISSUE THREAD

**Issue by pdurbin** [investinopen/InvestInOpen#1]
https://github.com/investinopen/InvestInOpen/blob/7a0ab03b3e3dead990cbac1cb2913a26f0e4a234/docs/statement0.2.md shows a very similar "it is clear" line twice in the same document. It makes for a strange read. 😄 

## First "it is clear" (under "Preamble, The Why")

"It is clear that the needs of today’s diverse scholarly communities are not being met by the existing largely uncoordinated scholarly infrastructure, which is dominated by vendor products that take ownership of the scholarly process and data."

## Second "it is clear" (under "The Issue")

"It is clear that the needs of today’s diverse scholarly communities are not being met by the existing scholarly infrastructure, which is dominated by vendor products that take ownership of the scholarly process and data as well as by North Atlantic dominance and digital colonialism."

p.s. I let the Dataverse community know about your webinar next week: https://groups.google.com/d/msg/dataverse-community/-N_wJTo2xrM/gXcZX7L-BQAJ 😄

**Comment by vsoch** [comment]
I think the first is trying to say that the needs of the academic community (and similar) don't fit well within a profit making model, and so there aren't tools developed optimally for them. As a result, we use a hodge-podge of solutions that don't support reproducibility. Thus, we need to have an initiative for open infrastructure to develop these tools collaboratively, taking into account the unmet needs of the community. 

I'm not sure what "North Atlantic dominance and digital colonialism" is, but it sounds pretty cool :)

**Comment by pdurbin** [comment]
@vsoch here's a screenshot of how it looks right now. I've highlighted the almost identical duplicate lines in grey. It's a bug. 😄 I can only assume it's unintentional. Nobody means to repeat themselves in this way. I mean nobody means to repeat themselves in this way.

And yes I'm wondering about North Atlantic dominance as well. 😄 

![Invest_in_Open_Infrastructure_A_Concept_0 2_Invest_in_Open_Infrastructure_-_2019-05-24_20 22 25](https://user-images.githubusercontent.com/21006/58362185-56394780-7e62-11e9-9821-55002871b92a.png)

**Comment by vsoch** [comment]
Ah, so your issue is with the repetition, not the statement? Why not open a PR with a suggestion to fix it?

**Comment by pdurbin** [comment]
Yes, my issue is with the repetition. I was an English tutor in college. This is pretty basic stuff. I'm sure it was unintentional. And it's probably easily fixed but I'll let someone else reword it.
